### PR TITLE
Alibaba Cloud for ImageNet resnet50 inference on ecs.gn5i-c8g1.2xlarge

### PR DESCRIPTION
**Pull request by pengli**

### CODE REVIEW

File: ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "v1.0",
+    "author": "Perseus AI Cloud Acceleration team in Alibaba Cloud",
+    "authorEmail": "patrick.lp@alibaba-inc.com, youliang.yl@alibaba-inc.com",
+    "framework": "TensorFlow 1.12.2",
+    "codeURL": "https://github.com/ali-perseus/DAWNBench_Inference",
+    "commitHash": "da18269a38a12e4e6f517e3d2caab9c330225204",
+    "model": "ResNet50",
+    "hardware": "Alibaba Cloud [ecs.gn5i-c8g1.2xlarge]",
+    "latency": 2.27,
+    "cost": 1.54e-06,
```
Comment (codyaustun) [inline ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json]: Can you point me to the website where I can find the price for this instance on Alibaba Cloud?

Comment (pengli) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json]: Hello, the price information is available at https://cn.aliyun.com/price/product?spm=5176.8789780.1092586.1.2d0257a8rKsPw1#/ecs/detail

please scroll down and find the instance with the name of "ecs.gn5i-c8g1.2xlarge".
You will see there are 9 columns, and the price of on-demand is shown in the fourth column.

I am sorry that the price page is in Chinese. I have made a translation for the titles of all these 9 columns as below,
Column 1: vCPU
Column 2: Type of instances.
Column 3: Memory.
Column 4: On-demand price (per hour).
Column 5: Monthly price.
Column 6: Monthly reduced price.
Column 7: Yearly price.
Column 8: Monthly price for 3 years.
Column 9: Monthly price for 5 years.

Please note: the price listed is in Chinese Yuan (CNY), please convert it to US dollar by an exchange rate of 0.15

Comment (terminatorzwm) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json]: Hi, I'm from Huawei ModelArts Team, I noticed that you'are using the model we comitted two months ago casue the MD5 values are the same. And your benchmark script is based on the one we comitted. So  I think you should at least give a statement about these information in your commit info. thanks.

Comment (pengli) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json]: @terminatorzwm , Hi, the benchmark script is based on  original  code https://github.com/tensorflow/tensorrt/blob/master/tftrt/examples/image-classification/image_classification.py, with modification to make it smaller and cleaner.

It is correct  that the resnet50 model is from ModelArts,  I will add this info in the commit.

Comment (codyaustun) [reply ImageNet/inference/AlibabaCloud_resnet50_1p4_ecs_tensorflow19.json]: @pengli I think you reported the wrong cost. It looks like you copied the cost from your [previous submission](https://github.com/stanford-futuredata/dawn-bench-entries/pull/66). The current cost is off by a factor 4.218/ 2.27 and should be ~8.28e-07.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Cost is incorrect. Please update or remove.

**Comment by pengli** [comment]
@codyaustun , sorry for the late response, cost is updated according to inference latency and US dollar/RMB exchange rate (0.1443). inference cost is 7.96e-7, the  calculation is (1000 * 3600) / 2.27 * 7.96e-7 / 0.1443 = CNY 8.75

**Review by codyaustun — APPROVED** [review]
Looks good to me! Thanks for fixing the cost.

**Pull request merged**